docs/_documentations/troubleshooting.md

```diff
@@ -504,7 +504,25 @@ Issue link: https://github.com/eclipse/codewind-docs/issues/64
 ## Node.js and Swift templates remain in the starting state
 The templates `Appsody Node.js template` and `Appsody Swift template` remain in the starting state by default because these templates do not have a server implemented, and therefore, its status cannot be detected. These templates do not have a server and are intended to help you implement your own server.
 
-**Workaround** To get the application into a **Started** state, use a server for the application. After the application has a server, Codewind can monitor the server, and the status turns to **Started** if the server is running. Alternatively, you can also temporarily [stop Codewind from continueously pinging the application](#how-to-stop-the-app-from-continuously-pinging).
+**Workaround** To get the application into a **Started** state, use a server for the application. After the application has a server, Codewind can monitor the server, and the status turns to **Started** if the server is running. Alternatively, you can also temporarily [stop Codewind from continuously pinging the application](#how-to-stop-the-app-from-continuously-pinging).
```

Comment:
If Codewind automatically monitors the server, then I think we should say, `After the application has a server, Codewind monitors the server, and the status turns...` instead of, `Codewind can monitor.` 

Because if Codewind automatically monitors the server, it's not really a matter as to whether it can or can't.

Reply:
ok